bin/cfupdate

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Might be better to have this as `module-update`. Now it implies that we only do this when working with cf which is not 100% true.